src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -153,17 +155,36 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
       !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))
     ) {
       element.classList.add("state-card-dialog");
-      element.addEventListener("click", () => this._handleClick(entityConf));
     }
 
     return html`
-      <div>${element}</div>
+      <div
+        @ha-click=${this._handleClick}
+        @ha-hold=${this._handleHold}
+        @ha-dblclick=${this._handleDblClick}
+        .longPress=${longPress({
+          hasDoubleClick: hasDoubleClick(entityConf.double_tap_action),
+        })}
+        .config=${entityConf}
+      >
+        ${element}
+      </div>
     `;
   }
 
-  private _handleClick(entityConf: EntitiesCardEntityConfig): void {
-    const entityId = entityConf.entity;
-    fireEvent(this, "hass-more-info", { entityId });
+  private _handleClick(ev: MouseEvent): void {
+    const config = (ev.currentTarget as any).config as any;
+    handleClick(this, this.hass!, config, false, false);
+  }
+
+  private _handleHold(ev: MouseEvent): void {
+    const config = (ev.currentTarget as any).config as any;
+    handleClick(this, this.hass!, config, true, false);
+  }
+
+  private _handleDblClick(ev: MouseEvent): void {
+    const config = (ev.currentTarget as any).entityConf as any;
```

Comment:
```suggestion
    const config = (ev.currentTarget as any).config as any;
```

Reply:
uh oh...I think I copied this from somewhere. Should double check the double tap PR, probably...